schema/dmt/compile.go

```diff
@@ -78,11 +78,20 @@ func todoAnonTypeName(nameOrDefn TypeNameOrInlineDefn) string {
 	case defn.TypeDefnList != nil:
 		defn := defn.TypeDefnList
 		return fmt.Sprintf("List__%s", todoAnonTypeName(defn.ValueType))
+	case defn.TypeDefnLink != nil:
+		return todoAnonLinkName(*defn.TypeDefnLink)
 	default:
 		panic(fmt.Errorf("%#v", defn))
 	}
 }
 
+func todoAnonLinkName(defn TypeDefnLink) string {
```

Comment:
does the `todo` prefix here mean this isn't done?

Reply:
I don't actually know .. there's a few of these in compile.go now and it's not entirely clear why the "todo" is in their names.. Do you think I should remove it from this one and `todoAnonTypeName()`? By adding links we should be making it complete, so maybe it's as simple as that.

Reply:
it seems reasonable to remove the todo's at this point. they seem misleading

Reply:
yeah, done